feathr_project/test/test_feature_registry.py

```diff
@@ -62,7 +64,33 @@ def test_feathr_register_features_e2e():
                                 output_path=output_path)
     client.wait_job_to_finish(timeout_sec=900)
 
+def test_feathr_register_features_partially():
```

Comment:
Could you elaborate the use case in the PR description? And it would be helpful to explain the exact meaning of "register features partially".

Reply:
Updated the PR description. Comments has been added to this test function.